package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "ill-watch-ui",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "apollo-boost": "^0.1.16",
+    "d3": "^5.7.0",
+    "graphql": "^14.0.2",
+    "react": "^16.5.1",
+    "react-apollo": "^2.1.11",
+    "react-dom": "^16.5.1",
+    "react-scripts": "1.1.5"
```

Comment:
Please remove `^` everywhere
Why:
https://bytearcher.com/articles/semver-explained-why-theres-a-caret-in-my-package-json/